src/components/TodoInsert.tsx

```diff
@@ -0,0 +1,89 @@
+import { BsPlusCircle } from 'react-icons/bs';
+import { useState } from 'react';
+import styled from 'styled-components';
+import {IHandleTodoInsert} from '../interface'
+
+interface ITodoInsertProps {
+  handleTodoInsert: IHandleTodoInsert;
+}
+
+const TodoInsert = ({ handleTodoInsert }: ITodoInsertProps) => {
+  const [inputValue, setInputValue] = useState('');
+  const handleSubmit = (e: React.FormEvent) => {
```

Comment:
```suggestion
  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
```
이벤트 대상을 명시해주시는게 좋아요. `React.FormEvent` 라는게 `<form>`의 모든 구성요소에서 일어나는 이벤트의 총 집합이기 때문에 살짝 명확하지 못한 감이 있죠.

Reply:
짚어 주셔서 감사합니다. 더 찾아보고 리팩토링 해보도록 하겠습니다!